api/v1/swagger.yaml

```diff
@@ -9222,6 +9224,11 @@ paths:
                 maxLength: 256
                 description: >
                   First and last name.
+              fullName:
+                type: string
+                maxLength: 60
+                description: >
+                  The user's full name.
```

Comment:
Should we make the maxLength of the fullName >= the maxLength of the username on L9222 ? Same comment for the other swagger changes.

Reply:
From my understanding I made the maxLength 60 because this is the max length of the fullName string in the model. Do you think this isn't long enough?

Reply:
60 is good too, just that when I looked at this PR  it kind of felt odd to see a full name field having a length that is less than the username field.  If there are other important things for you to do, we can talk about this later.  I have no problems with approving this PR. Let me know.